backend/test/UserTest.js

```diff
@@ -292,7 +281,7 @@ describe('user', () => {
           });
       });
       it('Sign up with short password', (done) => {
-        const request = { login: 'Piska', email: 'Niita@ad.erd', password: 'Password12' };
+        const request = { login: 'Piska', email: 'Niita@ad.erd', password: 'Password' };
```

Comment:
👎

Reply:
yee,i know,i will give one day for myself and fix all stupid variables